lib/ranked-model/ranker.rb

```diff
@@ -73,6 +73,10 @@ def position
         instance.send "#{ranker.name}_position"
       end
 
+      def relative_rank
+        finder.unscope(:select).where("#{ranker.column} < #{rank}").count(:all)
```

Comment:
You may not need to `.unscope(:select)` anymore since you're using `count(:all)`. Can you test this and confirm?

Reply:
Yes, we would need this `unscope(:select)`

We would have required it in case we were just doing `count`. Will update this.

Reply:
Just double-check that, as I tested it on this end and `count(:all)` obliterates the select anyway and replaces it with `SELECT COUNT(*)`. The SQL query was the same with and without `.unscope(:select)`. In short, I think you can safely omit that part.

Reply:
I had double checked before making the change. `count(:all)` converts to `SELECT COUNT(*)`

Reply:
Sorry, yes I misunderstood what you wrote earlier :)